examples/0_basic/0_create_did.rs

```diff
@@ -21,10 +20,25 @@ use iota_client::secret::SecretManager;
 use iota_client::Client;
 
 /// Demonstrates how to create a DID Document and publish it in a new Alias Output.
+///
+/// This example can be easily adapted to run in a private tangle by changing
+/// the network_endpoint and faucet_endpoint. In this example we use
+/// the Shimmer Testnet API.
+///
+/// A private tangle can be run by following the instructions in
+/// https://github.com/iotaledger/hornet/tree/develop/private_tangle.
```

Comment:
I've been told to avoid the "tangle" terminology.

Should we link to the Wiki instead of GitHub?

```suggestion
/// In this example we connect to the Shimmer testnet, but it can be adapted 
/// to run on a private network by setting the network and faucet endpoints. 
///
/// See the following instructions on running your own private network 
/// https://wiki.iota.org/hornet/develop/how_tos/private_tangle
```